en/rocky/8/guides/RL_rsnapshot_backup.md

```diff
@@ -0,0 +1,361 @@
+
+```
+echo 1441 > /var/run/rsnapshot.pid 
+mkdir -m 0755 -p /mnt/backup/storage/beta.0/ 
+/usr/bin/rsync -a --delete --numeric-ids --relative --delete-excluded \
+    /home/ /mnt/backup/storage/beta.0/localhost/ 
+mkdir -m 0755 -p /mnt/backup/storage/beta.0/ 
+/usr/bin/rsync -a --delete --numeric-ids --relative --delete-excluded /etc/ \
+    /mnt/backup/storage/beta.0/localhost/ 
+mkdir -m 0755 -p /mnt/backup/storage/beta.0/ 
+/usr/bin/rsync -a --delete --numeric-ids --relative --delete-excluded \
+    /usr/local/ /mnt/backup/storage/beta.0/localhost/ 
+touch /mnt/backup/storage/beta.0/
+```
+
+Once you are satisfied with the test, go ahead and run it manually the first time without the test:
+
+`rsnapshot -c /etc/rsnapshot.conf beta`
+
+When the backup finishes, navigate to /mnt/backup and take a look at the directory structure that was created there. There will be a `storage/beta.0/localhost` directory, followed by the directories that you specified to backup. 
+
+### Further Explanation
+
+Each time the backup is run, it will create a new beta increment, 0-6, or 7 days worth of backups. The newest backup will always be beta.0 where as yesterday's backup will always be beta.1. The size of each of these backups will appear to take up the same amount (or more) of disk space, but this is because of rsnapshot's use of hard links. To restore files from yesterday's backup, you would simply copy them back from beta.1's directory structure. Each backup is only an incremental backup from the previous run, BUT, because of the use of hard links, each backup directory, contains the file, or the hard-link to the file in whichever directory it actually was backed up in. So to restore files, you don't have to pick and choose which directory or increment to restore them from, just what time stamp the backup should have. It's a great system and uses far less disk space than many other backup solutions.
+
+### Setting The Backup To Run Automatically
+
+Once everything has been tested and we know that things will work without issue, the next step is to set up the crontab for root's so that all of this can be done automatically every day:
+
+`sudo crontab -e`
+
+If you haven't run this before, choose vim.basic as your editor or your own editor preference when the `Select an editor` line comes up. 
+
+We are going to set our backup to automatically run at 11 PM, so we will add this to the crontab: 
+
+```
+## Running the backup at 11 PM
+00 23 *  *  *  /usr/bin/rsnapshot -c /etc/rsnapshot.conf beta`
+```
+
+## Multiple Machine or Multiple Server Backups
+
+Doing backups of multiple machines from a machine with a RAID array or large storage capacity, either on premises or from across the Internet works very well. If running from across the Internet, you need to make sure that both locations have adequate bandwidth for the backups to occur. In my previous job, we used both an on-premise disk array and an off-premise disk array for maximum redundancy. 
```

Comment:
Maybe revise the last sentence to: "You can use rsnapshot to synchronize an on-site server with an off-site backup array or backup server to improve data redundancy."